packages/appium/docs/en/guides/managing-exts.md

```diff
@@ -2,4 +2,78 @@
 title: Managing Appium Drivers and Plugins
 ---
 
-TODO
+To do anything useful with Appium, you need to have at least one [Driver](../intro/drivers.md)
+installed, otherwise Appium won't know how to automate anything. And there is a whole
+[Ecosystem](../ecosystem/index.md) of drivers and plugins out there which enable you to get your
+job done with Appium. This guide helps explain how to manage these drivers and plugins. There are
+two basic strategies: using Appium's extension CLI interface, or managing extensions yourself in an
+NPM-based project.
+
+## Using Appium's Extension CLI
+
+With Appium's [Extension CLI](../cli/extensions.md), you let Appium manage drivers and plugins for
+you, and simply use CLI commands to tell Appium which extensions you would like to install, update,
+or remove. Here's an example of how you might install a driver using the CLI:
+
+```bash
+appium driver install xcuitest
+```
+
+This simple command will install the latest version of the [XCUITest
+Driver](https://github.com/appium/appium-xcuitest-driver). The Extension CLI comes with a variety
+of commands and parameters; see the documentation for that command for all the specifics.
+
+The all-important question when Appium is managing your extensions for you is: where do they live?
+Appium manages extensions in a directory specified by the `APPIUM_HOME` environment variable. You
+can set that variable to anything you like, and Appium will manage its extensions there. You can
+therefore also use the `APPIUM_HOME` environment variable to manage different sets of extensions,
+for example if you want to have the same driver installed at conflicting versions:
+
+```bash
+APPIUM_HOME=/path/to/home1 appium driver install xcuitest@4.11.1
+APPIUM_HOME=/path/to/home2 appium driver install xcuitest@beta
```

Comment:
Actually xcuitest does not have `beta` channel, so probably the `beta` can be another version such as `4.12.1`. Maybe such different versions should be enough as this context

Reply:
yeah ok, i guess it should be a legitimate version